cogs/fun.py

```diff
@@ -95,9 +96,59 @@ async def dictionary(self, ctx: commands.Context, *, word: str):
             embed.add_field(name="Adjective", value=adjectives, inline=False)
         if verbs:
             embed.add_field(name="Verb", value=verbs, inline=False)
-            
+
         await ctx.send(embed=embed)
 
+    def getSynOrAnt(self, method, word):
+        methods = {
+            "synonym" : Synonyms(search_string=word).find_synonyms(),
+            "antonym" : Antonyms(search_string=word).find_antonyms()
+        }
+        if "Please verify that the word is spelled correctly." in methods[method]:
+            raise Exception
+        return methods[method]
+
+    def fillEmbed(self, title, word, results):
+        embeds = []
+        fields = []
+        field = []
+        max_char = 1024
+        for index, result in enumerate(results):
+            max_char -= len(result) + 2
+            if max_char > 0:
+                field.append(result)
+                if index == len(results) - 1:
+                    fields.append(field)
+                    field = []  
+            else:
+                fields.append(field)
+                field = []
+                max_char = 1024
+        for field in fields:
+            page = discord.Embed(title=f"Zen | Thesaurus")
+            page.add_field(name="Word", value=f"{word.title()}", inline=False)
+            page.add_field(name=f"{title}", value=f"{', '.join(field)}", inline=False)
+            embeds.append(page)     
+        return embeds
```

Comment:
Could move these functions above all the commands functions to make this file look cleaner.

Reply:
Will do!